db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/StaticMethodCallNode.java

```diff
@@ -704,6 +691,100 @@ private void resolveRoutine(FromList fromList,
 
 				break;
 			}
+
+			// If the resolved StaticMethodCallNode has the LANGUAGE PYTHON,
+			// it will resolves in calling method PyProcedureWrapper
+			// The parameters, and retruen type for these two Stored Procedures are the same
+			if(this.routineInfo!=null && this.routineInfo.getLanguage().equals("PYTHON"))
+			{
+
+				this.resolvePyProc = true;
+				// need to add compiled Python code bytes into signature and methodParms
+				byte[] compiledPyCode = this.routineInfo.getCompiledPyCode();
+
+				// Update the parameterCnt ,parameterModes, parameterNames, parameterTypes
+				int origParmCnt = this.routineInfo.getParameterCount();
+				int updatedParmCnt = origParmCnt + 1;
+
+				int[] updatedParmModes = new int[updatedParmCnt];
+				String[] updatedParmNames = new String[updatedParmCnt];
+				TypeDescriptor[] updatedParmTypes = new TypeDescriptor[updatedParmCnt];
+				if(origParmCnt > 0) {
+					System.arraycopy(this.routineInfo.getParameterModes(), 0, updatedParmModes,0,origParmCnt);
+					System.arraycopy(this.routineInfo.getParameterNames(), 0, updatedParmNames,0,origParmCnt);
+					System.arraycopy(this.routineInfo.getParameterTypes(), 0, updatedParmTypes, 0, origParmCnt);
+				}
+				updatedParmModes[origParmCnt] = 1; // States the input script is an IN parameter
+				// Here Need to use a unique name
+				updatedParmNames[origParmCnt] = "PYCODE";
+				// I am not sure whether the format Id is correct or not
+				updatedParmTypes[origParmCnt] = new TypeDescriptorImpl(new BaseTypeIdImpl(442),
```

Comment:
I assume 442 is from the BLOB type defined in StoredFormatIds.java, could you use BLOB_TYPE_ID_IMPL to make it more readable?

Reply:
Yes, I was experiencing some issue with using BLOB_TYPE_ID_IMPL as it resulted in 440 (or some other int) and hence used a hardcoded int instead. Then I forgot to figure out what caused the problem.

I will replace it with BLOB_TYPE_ID_IMPL.